src/ftlbestemmelser.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Kodeverk/Folketrygdloven_kap2_bestemmelser
+ * @module
+ */
+const ftlbestemmelser = [
```

Comment:
I Confluence heter kodeverket `Folketrygdloven_Kapittel_2_Medlemskap`. Jeg synes det var litt langt, og `ftlbestemmelser` litt kort 😛 . 
Ser det står en kommentar på Confluence som tyder på at de ikke har bestemt navnet helt. Kanskje verdt å høre videre?

Reply:
Jeg lurer også på om vi ikke bør ha et felles interface `Bestemmelse` e.l., som `LovvalgBestemmelse` er i java https://github.com/navikt/melosys-kodeverk-java/blob/master/melosys-kodeverk-generator/src/main/resources/application.yml
Bedre løsning ville jo vært å gi backend ansvaret for kodeverket. Men det får vi nok ikke til 😛

Reply:
Den het opprinnelig ftlkap2bestemmelser, men yarn build funkerte dårlig med tall i navnet..

Ser nå etter et avklaringsmøte at akkurat dette kodeverket ikke trengs helt enda, blir en del av stegvelgeren så det kan være at det kan tas ut og legges i en egen branch inntil videre